source/includes/changelogs/releases/100.7.2.rst

```diff
@@ -0,0 +1,13 @@
+.. _100.7.2-changelog:
+
+100.7.2 Changelog
+-----------------
+
+*Released 2023-05-30*
+
+This release fixes an issue with installing Database Tools on RHEL aarch64 architecture. 
+
+Bug
+~~~
+
+- :issue:`TOOLS-3309` Fix release issue on RHEL ARM architecture
```

Comment:
Should it be "AArch64" to match line 8?

Reply:
I'll leave this question open for the external reviewer.

Reply:
I agree. I'll change TOOLS-3309 accordingly.

Reply:
Fixed here as well. Thanks!